test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -212,8 +219,105 @@ private static void dumpProcessLog(final ProcessBuilder b) {
 		catch (FileNotFoundException e) {
 			System.err.println("Attempted to dump the log file '" + log.getAbsolutePath() + "' but it was not found.\n" + e.getMessage());
 		}
+
+	private static int countStringInFile(String stringToLookFor, File file1) throws FileNotFoundException,IOException {
+		int count = 0;
+		String line = null;
+		String[] buffer;
+
+		BufferedReader bfReader = new BufferedReader(new FileReader(file1));
+		while((line = bfReader.readLine()) != null)   {
+			if (line.contains(stringToLookFor))
+				count++;
+		}
+		logger.info("File: "+file1.getName()+" has "+count+" occurences of the word "+stringToLookFor);
+		bfReader.close();
+		return count;
+        }
+
+	private static boolean checkLogFiles(String fileNamePattern, String stringToLookFor) {
+		boolean foundMatch = false;
+		try{
+			File f = new File(System.getProperty("user.dir"));
+			FilenameFilter filter = new FilenameFilter() {
+				public boolean accept(File f, String name){
+					return name.startsWith(fileNamePattern);
+				}
+			};
+
+			//List all the log files matching the criteria in the order of last modified
+			File[] files = f.listFiles(filter);
+			Arrays.sort(files, Comparator.comparingLong(File::lastModified).reversed());
```

Comment:
Is it ok to check all vlog files? Is it possible that there are some vlogs files left over from a previous test? Maybe not if each test creates its own directory. It's more robust though if we delete any vlog files when starting the test.

Reply:
Each test creates its own directory and sub-directory. And within each directory each test creates its own verbose logs and other logs with the name of the test prefix to the log files. The test I added is named `testServerSCC` ,so in my code I only read the client verbose logs created by my test with the following code: `if(!checkLogFiles("testServerSCC.client.jitverboselog", "remote deserialized"))`

The current code does not delete any log files created by the test. Possibly for diagnostic purposes after the tests are completed. Hence I did not touch the existing logic.

Reply:
ok, though to be more robust I would change the code to test exactly one vlog, the one produced by the second client instance. The PID of the JVM that produced the vlog is attached as a suffix, so if you retrieve the PID of the second client JVM you should be able to narrow down to just one vlog.

Reply:
Added the code to check the single log file.